app/models/firm.rb

```diff
@@ -85,6 +85,28 @@ def subsidiary?
     parent.present?
   end
 
+  def field_order
+    [
+      :email_address,
+      :telephone_number,
+      :address_line_one,
+      :address_line_two,
+      :address_town,
+      :address_county,
+      :address_postcode,
+      :in_person_advice_methods,
+      :free_initial_meeting,
+      :initial_meeting_duration,
+      :initial_advice_fee_structures,
+      :ongoing_advice_fee_structures,
+      :allowed_payment_methods,
+      :minimum_fixed_fee,
+      :percent_total
+    ] +
```

Comment:
Nice job. Maybe from here though we could try:

```ruby
].tap do |ordered_fields|
  ordered_fields << I18n.t(...).keys
  ordered_fields << :investment_sizes
end
```

That's stylistic nit-picking though, but otherwise it just doesn't look particularly 'pretty' ruby, if that makes any sense at all :smile: